### PR TITLE
Make solana ping take optional lamports argument

### PR DESCRIPTION
**Pull request by ryoqun**
No strong usecase, but it's handy to able to set arbitrary number of lamports to send when `ping`-ing.

Particularly in my case, I needed this for 0 lamport while working on #6890 to create consistent flow of TXs avoiding some oneliners.

### CODE REVIEW

File: cli/src/cluster_query.rs [hunk 1]
```diff
@@ -84,6 +84,13 @@ impl ClusterQuerySubCommands for App<'_, '_> {
                         .takes_value(true)
                         .help("Stop after submitting count transactions"),
                 )
+                .arg(
+                    Arg::with_name("lamports")
+                        .long("lamports")
+                        .value_name("NUMBER")
+                        .takes_value(true)
+                        .help("Number of lamports to transfer for each transaction [default: 1]"),
```
Comment (mvines) [inline cli/src/cluster_query.rs]: By adding `.default_value("1")` here, you can remove the explicit "[default: 1]"

File: cli/src/cluster_query.rs [hunk 2]
```diff
@@ -128,6 +135,11 @@ pub fn parse_catchup(matches: &ArgMatches<'_>) -> Result<CliCommandInfo, CliErro
 }
 
 pub fn parse_cluster_ping(matches: &ArgMatches<'_>) -> Result<CliCommandInfo, CliError> {
+    let lamports = if matches.is_present("lamports") {
+        value_t_or_exit!(matches, "lamports", u64)
+    } else {
+        1
+    };
```
Comment (mvines) [inline cli/src/cluster_query.rs]: Adding `.default_value("1")` allows this to be simplified down to `let lamports = value_t_or_exit!(matches, "lamports", u64);` too

File: cli/src/cluster_query.rs [hunk 3]
```diff
@@ -84,6 +84,13 @@ impl ClusterQuerySubCommands for App<'_, '_> {
                         .takes_value(true)
                         .help("Stop after submitting count transactions"),
                 )
+                .arg(
+                    Arg::with_name("lamports")
+                        .long("lamports")
```
Comment (ryoqun) [inline cli/src/cluster_query.rs]: In courtesy of the `ping`, I wondered what corresponding short options is, but couldn't determine. So, there is no `short` by intention. Maybe `ping`'s `-p pattern` or `-s packetsize` will be the closest counterpart?

Comment (mvines) [reply cli/src/cluster_query.rs]: No short option here is fine with me.

Comment (ryoqun) [reply cli/src/cluster_query.rs]: I see! Me too.

File: cli/src/cluster_query.rs (hunk 1 above)
Comment (ryoqun) [reply cli/src/cluster_query.rs]: Heh, too hurrying me. I just couldn't notice the API usage around the code... 6fc7bcde24b5d42ac26cbf63d77288d8ac5de90b

File: cli/src/cluster_query.rs (hunk 2 above)
Comment (ryoqun) [reply cli/src/cluster_query.rs]: same as https://github.com/solana-labs/solana/pull/7029#discussion_r347735823!

**Comment by ryoqun** [comment]
@mvines Once the CI passes (I'm pretty sure), I'll merge this! Thanks quick review!

**Pull request merged**